.azure-pipelines/template-commonlib.yml

```diff
@@ -39,9 +39,10 @@ jobs:
       displayName: Make configure
     - script: |
         set -ex
+        grep amd64 .arch && extarget=target/docker-sonic-vs.gz
         for bldenv in $(bldenvs)
         do
-          SONIC_BUILD_JOBS=$(nproc) BLDENV=$bldenv make -f Makefile.work lib-packages ENABLE_DOCKER_BASE_PULL=y
```

Comment:
Why remove the option ENABLE_DOCKER_BASE_PULL=y

Reply:
make configure step has already pulled docker base image.